### ISSUE TITLE
Server events causing booklet throw error : Unable to get property 'left' of undefined or null reference

### ISSUE THREAD

**Issue by swapnatv** [builtbywill/booklet#36]
First of all thank u so much for such a great plugin. I have tried using this booklet plugin to load dynamic contents using Ajax. Need your help to solve one issue. The issue is that whenever there is a server side event occurring(say button click), I get an error:- Error: Unable to get property 'left' of undefined or null reference
 Error is thrown in booklet.latest.min.js

 {aB.draggable({axis:"x",containment:[aH.offset().left,0,aD.offset().left+K-a1,y],drag:function

I am using master page. Jquery reference is thus made in master page only.Have downloaded the latest version of booklet also and tried. Please help if possible!
